src/input/input/windows.rs

```diff
@@ -273,6 +273,7 @@ impl Iterator for SyncReader {
 pub struct AsyncReader {
     event_rx: Receiver<InputEvent>,
     shutdown: Arc<AtomicBool>,
+    thread: Option<thread::JoinHandle<()>>,
```

Comment:
Why is this an option?

Reply:
`JoinHandle::join` takes `self`, and `stop` `&mut self` since it's called on drop. This is the easiest way.